sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -66,6 +66,23 @@ evaluate(
 )
 ```
 
+- Simulator now requires configuration to call the prompty within the `azure_ai_project`. This enables key-less usage of the simulator.
```

Comment:
this entry probalby needs some rewording 😄

Reply:
Done